### DISCUSSION TITLE
aliyundrive-webdav尚未支持ipv6

### DISCUSSION THREAD

**Discussion by liuchenger** [messense/aliyundrive-webdav#753] [Q&A] [answered]
### 问题描述

大佬，能把aliyundrive-webdav增加一下ipv6访问支持吗，我家里只有ipv6公网，非常喜欢你的这个项目，期待你的回复

### 重现步骤

无

### 版本

2.0.3

### 运行平台

docker arm64

### 日志

无

**Comment by liuchenger** [comment, marked as answer]
大佬看到问题后回复一下，我测试着大部分容器都是支持ipv6的，听别人说作者可以做成支持或者不知道ipv6，所以希望大佬可以给咱们项目增加ipv6的支持

**Reply by messense** [reply]
你是不会搜索吗：https://github.com/messense/aliyundrive-webdav/discussions/159

**Reply by liuchenger** [reply]
-e HOST=:: \
加上一行这个可以了，多谢大佬提醒
